### PR TITLE
scheduler yaml + op checkpoint

### PR DESCRIPTION
**Pull request by cfarre**

### CODE REVIEW

**Comment by ferantivero** [comment]
just wanted to mentioned that it looks like we are about to get rid of the integration w/ l5d from scheduler.

**Comment by francischeung** [comment]
This is a temporary fix to get the scheduler to be able to access the other services. For some reason putting all shipping related services in the bc-shipping namespace instead of the default namespace causes 502 Bad Gateway errors.

I verified that when I move our services back into the default namespace, everything works fine.

May need to alter the l5d dtab.

Fernando, can you please investigate?
________________________________

**Comment by ferantivero** [comment]
I know what's going on in here. Please let me take care of this.

**Comment by ferantivero** [comment]
As @francischeung has mentioned [this l5d configuration](https://raw.githubusercontent.com/linkerd/linkerd-examples/master/k8s-daemonset/k8s/servicemesh.yml) is able to discover services in default-ns-only.

I've just created an issue to address this: https://github.com/mspnp/microservices-reference-implementation/issues/69

File: k8s/scheduler.yaml [hunk 1]
```diff
@@ -80,18 +82,7 @@ spec:
...
-            valueFrom:
-              fieldRef:
-                fieldPath: spec.nodeName
-          - name: POD_IP
-            valueFrom:
-              fieldRef:
-                fieldPath: status.podIP
-          - name: http_proxy
-            value: $(NODE_NAME):4140	
+            value: "32"
```
Comment (francischeung) [inline k8s/scheduler.yaml]: Please include the l5d configuration but commented out.

**Comment by cfarre** [comment]
the problem is related with service discovery. Deploying services in different namespace causes l5d not to work. Not only for scheduler. i tried to hit ingestion service through l5d proxy and it fails to discover ingestion service.

if i deploy  ingestion service in default namespace it works.



Sent from my Verizon, Samsung Galaxy smartphone


-------- Original message --------

**Pull request merged**